sys/linux/bpf_lsm.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/bpf.h>
+
+resource bpf_lsm_btf_id[bpf_btf_id]
+
+syz_btf_id_by_name$bpf_lsm(name ptr[in, string[bpf_lsm_func_names]]) bpf_lsm_btf_id (timeout[3000], prog_timeout[3000])
+
+bpf$BPF_LSM_PROG_LOAD(cmd const[BPF_PROG_LOAD], arg ptr[in, bpf_lsm_prog], size len[arg]) fd_bpf_prog (timeout[3000], prog_timeout[3000])
+
+type bpf_lsm_prog bpf_prog_t[const[BPF_PROG_TYPE_LSM, int32], const[BPF_LSM_MAC, int32], bpf_lsm_btf_id, const[0, int32]]
+
+# The list is generated by the following command:
+# bpftool btf dump file /sys/kernel/btf/vmlinux | sed -n "s/.*FUNC '\(bpf_lsm\w*\).*/\"\1\"/p" | tr '\n' ',' | sed 's/,$/\n/'
```

Comment:
We need to mention that it can be run against the vmlinux file on the host. And probably use "vmlinux" in the command, and mention that it can be run against /sys/kernel/btf/vmlinux in the target kernel as well.
Running on the host is fewer steps and easier (e.g. host-build bpftool won't necessary run inside of the VM due to different glibc or something else).